GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -34,6 +34,7 @@
 #import "GoogleSignIn/Sources/GIDAuthentication_Private.h"
 #import "GoogleSignIn/Sources/GIDGoogleUser_Private.h"
 #import "GoogleSignIn/Sources/GIDProfileData_Private.h"
+#import "GoogleSignIn/Sources/GIDUserAuth_Private.h"
```

Comment:
Add the public header above as well so that we're not relying on transitive includes.

Reply:
Doen.